### ISSUE TITLE
Is there a scoring method available from where we can get some model metrics

### ISSUE THREAD

**Issue by shuvayan** [pymc-labs/pymc-marketing#531]
Hello, 

For models built using sklearn we can do the following:

from sklearn.model_selection import cross_val_score, TimeSeriesSplit
print(cross_val_score(model, X, y, cv=TimeSeriesSplit()).mean())

To get some metrics like R2 , etc. 
Is there anything similar in pymc_marketing or pymc which can be used for model comparison.

**Comment by wd60622** [comment]
Each model you build should be able to be used in the pymc comparisons techniques shown [here](https://www.pymc.io/projects/docs/en/stable/learn/core_notebooks/model_comparison.html)

The sklearn metrics can be used on arbitrary numpy arrays. Therefore, you can use the [`predict`](https://www.pymc-marketing.io/en/stable/api/generated/pymc_marketing.mmm.delayed_saturated_mmm.DelayedSaturatedMMM.predict.html) in order to get the mean prediction for the model given the inputs. Two models predictions could be compared as well

What happens if you use that `cross_val_score` function on one of our models? There is the `fit` and `predict` methods which might be the requirement of using sklearn validation. Not sure though

**Comment by shuvayan** [comment]
Solved. The cross_val_score or even the r2_score from sklearn.metrics method works fine post fitting the model via the fit method and it also works with the prediction on test set.
Awesome, thank you so much!!

**Comment by shuvayan** [comment]
Thank you!!
